src/app/plugins/platform/Docker/plugins/cms/WordPress/templates/_phpmd.xml.dist.ejs

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<ruleset name="phpmd.xml"
+        xmlns="http://pmd.sf.net/ruleset/1.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd"
+        xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd">
+    <description>PHPMD Ruleset for a WordPress Project</description>
+    <rule ref="rulesets/cleancode.xml/BooleanArgumentFlag" />
+    <rule ref="rulesets/cleancode.xml/ElseExpression" />
+    <rule ref="rulesets/cleancode.xml/StaticAccess" />
```

Comment:
@slucero I need to look into it further but Timber is being called statically in a lot of cases so this is going to trip all of those cases up.

Reply:
Our best route forward here may be to get this generated configuration in place on a couple of existing WordPress projects and evaluate what Codacy is reporting to determine which rules need to be tweaked.

Reply:
I feel like we can add this to an existing project easy enough, like I did for Gesso WP, without baking it into the Web Starter. My concern is we're going to roll this out and it's going to get implemented as a snowflake/non-standard instance, or instances that people really don't understand.